### PR TITLE
ci: only collect coverage data in src

### PR DESCRIPTION
**Pull request by BusyJay**
@ngaut @siddontang PTAL

### CODE REVIEW

File: travis-build/Makefile [hunk 1]
```diff
@@ -21,11 +23,12 @@ ${ROCKSDB_DIR}/librocksdb.so: ${ROCKSDB_DIR}/libgflags.a
 	make shared_lib && \
 	cp librocksdb.so* ${ROCKSDB_DIR}
 
+# TODO: use offical archive once https://github.com/SimonKagstrom/kcov/pull/146 is resolved and published.
```
Comment (siddontang) [inline travis-build/Makefile]: The PR is merged, can update.

Comment (BusyJay) [inline travis-build/Makefile]: It's not released yet.

Comment (siddontang) [inline travis-build/Makefile]: you can use the master directly.

Comment (BusyJay) [inline travis-build/Makefile]: No, `use official archive` means use a stable version of official archive. So we should use a specific version or a version with a patch, latter is my very branch.

**Comment by siddontang** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**